python/src/deltachat/account.py

```diff
@@ -160,7 +161,7 @@ def _preconfigure_keypair(self, addr, public, secret):
         if res == 0:
             raise Exception("Failed to set key")
 
-    def update_config(self, kwargs):
+    def update_config(self, kwargs: Dict[str, Any]) -> None:
```

Comment:
I think this was a mistake and the intended behavior was:
 ```suggestion
    def update_config(self, **kwargs: Dict[str, Any]) -> None:
```

as it currently is users of the function have to wrap the `name=value` pairs inside a `dict()`